src/extension.ts

```diff
@@ -188,6 +188,7 @@ export function activate(context: ExtensionContext) {
     xml['xml']['java']['home'] = javaHome;
     xml['xml']['format']['trimFinalNewlines'] = workspace.getConfiguration('files').get('trimFinalNewlines', false);
     xml['xml']['format']['insertFinalNewline'] = workspace.getConfiguration('files').get('insertFinalNewline', false);
+    xml['xml']['format']['quoteStyle'] = workspace.getConfiguration('xml').get('preferences.quoteStyle', "double");
```

Comment:
The new preference for vscode-xml is `xml.preferences.quoteStyle`, but it is currently being mapped to `xml.format.quoteStyle` in order to make https://github.com/eclipse/lemminx/pull/669 simpler. 

Alternatively, I could remove this line, and on the lemminx side, create a new `XMLPreferences` class that contains `XMLPreferences#quoteStyle`.

Reply:
I'd rather stay consistent and have the XMLPreferences node on the server side

Reply:
Okay, just making sure before I commit to this, if I take that route, I would have to add XMLPreferences as a new parameter in places like:
https://github.com/eclipse/lemminx/blob/c06f71a737bf1f84f82145005acd12843c891ee1/org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/services/extensions/ICodeActionParticipant.java#L25-L26

and 

https://github.com/eclipse/lemminx/blob/c06f71a737bf1f84f82145005acd12843c891ee1/org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/utils/XMLBuilder.java#L34

I know in general, it's preferred to have a lower number of parameters. Is it fine to add a parameter in this case?

Reply:
Yes new pref

Reply:
and remove  `xml['xml']['format']['quoteStyle']`